Terminal Commands.md

```diff
@@ -22,8 +28,18 @@ Some commands can be found in the `bin` directory.
 `mkdir` - makes a directory<br/>
 `rmdir` - removes an **empty** directory<br/>
 
+`cp file1 file2` - used to copy file1 to file2.<br>
```

Comment:
For `cp` use something like "copies file at location loc1 to loc2" . Similarly for `mv`